node-tests/blueprints/dependencyScenarios.js

```diff
@@ -19,9 +15,16 @@ const scenarios = [
       npm: ['ember-cli-less']
     },
     dependencies: {
-      bower: {
-        bootstrap: null
-      },
+      npm: {
+        bootstrap: bs3Regex
+      }
+    }
+  },
+  {
+    installed: {
+      npm: ['ember-cli-less', 'bootstrap-sass']
```

Comment:
Yes, good catch. No idea what my thought was, probably just too late in the evening... :)